sdk/storage/azure-storage-file-share/tests/recordings/test_file.test_copy_file_with_specifying_acl_copy_behavior_attributes.yaml

```diff
@@ -137,51 +137,49 @@ interactions:
       content-md5:
       - yaNM/IXZgmmMasifdgcavQ==
       date:
-      - Tue, 28 Jan 2020 05:13:22 GMT
+      - Fri, 01 Apr 2022 02:17:42 GMT
       etag:
-      - '"0x8D7A3B0CBD74C93"'
+      - '"0x8DA1385CE26AC91"'
       last-modified:
-      - Tue, 28 Jan 2020 05:13:22 GMT
+      - Fri, 01 Apr 2022 02:17:43 GMT
       server:
       - Windows-Azure-File/1.0 Microsoft-HTTPAPI/2.0
       x-ms-request-server-encrypted:
       - 'true'
       x-ms-version:
-      - '2019-07-07'
+      - '2021-04-10'
     status:
       code: 201
       message: Created
 - request:
     body: null
     headers:
       Accept:
-      - '*/*'
+      - application/xml
       Accept-Encoding:
       - gzip, deflate
       Connection:
       - keep-alive
       Content-Length:
       - '0'
       User-Agent:
-      - azsdk-python-storage-file-share/12.1.0 Python/3.7.3 (Windows-10-10.0.18362-SP0)
+      - azsdk-python-storage-file-share/12.8.0b1 Python/3.10.2 (Windows-10-10.0.19044-SP0)
       x-ms-copy-source:
-      - https://emilystageaccount.file.core.windows.net/utsharedd3c1c70/filedd3c1c70
+      - https://vincenttrancanary.file.core.windows.net/utsharedd3c1c70/filedd3c1c70
       x-ms-date:
-      - Tue, 28 Jan 2020 05:13:22 GMT
+      - Fri, 01 Apr 2022 02:17:43 GMT
       x-ms-file-attributes:
       - Temporary|NoScrubData
-      x-ms-file-copy-ignore-read-only:
+      x-ms-file-copy-ignore-readonly:
```

Comment:
This is curious - did the header name change between API versions?
Or did something change in the swagger-generated layer?